<ui version="4.0" >
 <class>HLAWindow</class>
 <widget class="QWidget" name="HLAWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Network Connections</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Available Connections</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QListWidget" name="mNetworkListWidget" >
        <property name="toolTip" >
         <string>Select an existing network connection from the list. </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="mNewPushButton" >
          <property name="toolTip" >
           <string>Create a new network connection. </string>
          </property>
          <property name="text" >
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mEditPushButton" >
          <property name="toolTip" >
           <string>Edit an existing network connection. </string>
          </property>
          <property name="text" >
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mDeletePushButton" >
          <property name="toolTip" >
           <string>Delete the currently selected network connection. </string>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mConnectPushButton" >
          <property name="toolTip" >
           <string>Connect to the currently selected federation. </string>
          </property>
          <property name="text" >
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Current Connection</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QPushButton" name="mDisconnectPushButton" >
        <property name="toolTip" >
         <string>Disconnect from the currently selected federation.</string>
        </property>
        <property name="text" >
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mCurrentConnectionLineEdit" >
        <property name="toolTip" >
         <string>The current federation you are connected to.</string>
        </property>
        <property name="text" >
         <string>None</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mReconnectOnStartupCheckBox" >
        <property name="toolTip" >
         <string>Check this box if you want the Stealth Viewer to reconnect to the previous connection on startup.</string>
        </property>
        <property name="text" >
         <string>Reconnect On Startup</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>2</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mClosePushButton" >
         <property name="toolTip" >
          <string>Close this window. </string>
         </property>
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mNewPushButton</tabstop>
  <tabstop>mEditPushButton</tabstop>
  <tabstop>mDeletePushButton</tabstop>
  <tabstop>mClosePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mClosePushButton</sender>
   <signal>clicked()</signal>
   <receiver>HLAWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
